### PR TITLE
feat: Add isPlaying method

### PR DESCRIPTION
**Pull request by mamane10**

### CODE REVIEW

File: ios/Plugin/Plugin.swift [hunk 1]
```diff
@@ -175,6 +175,17 @@ public class NativeAudio: CAPPlugin {
         audioAsset.setVolume(volume: volume as NSNumber)
         call.success()
     }
+
+    @objc func isPlaying(_ call: CAPPluginCall) {
```
Comment (bazuka5801) [inline ios/Plugin/Plugin.swift:179]: Also add definition in Plugin.m file for new method

**Review by bazuka5801 — CHANGES_REQUESTED** [review]

File: ios/Plugin/Plugin.swift (hunk 1 above)
Comment (mamane10) [reply ios/Plugin/Plugin.swift:179]: I did as you asked. Thanks for your comment.

**Comment by indraraj26** [comment]
any plan to release isPlaying ?

File: ios/Plugin/AudioAsset.swift [hunk 2]
```diff
@@ -171,4 +171,14 @@ public class AudioAsset: NSObject, AVAudioPlayerDelegate {
     func playerDecodeError(player: AVAudioPlayer!, error: NSError!) {
         
     }
+
+    func isPlaying() -> Bool {
+        if channels.count != 1 {
+            return 0
```
Comment (indraraj26) [inline ios/Plugin/AudioAsset.swift:177]: here it should be `return false` it will through the error as expect return type should be of Boolean

**Review by indraraj26 — CHANGES_REQUESTED** [review]

File: ios/Plugin/AudioAsset.swift (hunk 2 above)
Comment (mamane10) [reply ios/Plugin/AudioAsset.swift]: Thanks, appreciate it

**Pull request merged**